style.css

```diff
@@ -0,0 +1,189 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+body {
+  background: #fff;
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  height: auto;
+}
+
+.top-nav {
+  display: flex;
+  overflow: hidden;
+  justify-content: space-between;
+  background-color: aqua;
+  position: fixed;
+  top: 0;
+  width: 100%;
+}
+
+.top-left {
+  display: flex;
+  list-style-type: none;
+  justify-content: space-evenly;
+  gap: 20px;
+}
+
+.center {
+  display: flex;
+  list-style-type: none;
+  justify-content: space-evenly;
+  gap: 20px;
+}
+
+.top-right {
+  display: flex;
+  float: right;
+  list-style-type: none;
+  justify-content: space-evenly;
+}
+
+.top-left li {
+  margin-inline: 20px;
+}
+
+.my-tube {
+  display: flex;
+  gap: 10px;
+}
+
+.fa-bars {
+  width: 50px;
+  font-size: 40px;
+  border: 0;
+}
+
+.fa-youtube {
+  width: 50px;
+  font-size: 35px;
+  border: 0;
+  color: #f50000;
```

Comment:
Kindly change the color to black. This will make our project have a nice look of color.